R/validateSettings.R

```diff
@@ -112,6 +115,25 @@ validateSettings <- function(data, settings, chart="eDish"){
   #valid=true if all checks pass, false otherwise
   settingStatus$valid <- settingStatus$checks%>%select(.data$valid)%>%unlist%>%all
   
+  #assess validity for each specified chart
+  settingStatus$charts <- list()
+  i<-1
+  for(chart in charts){
+    obj<-list(chart=chart)
```

Comment:
Just curious why you decided to structure this way...seems a little easier to extract chart-specific validation if under a named list  e.g. ``` settingStatus$charts[["edish"]] ```

Reply:
Change you proposed looks good to me.